src/tools/scamv/proggen/bir_prog_gen_randLib.sml

```diff
@@ -149,7 +149,7 @@ struct
      alphaList @ numList @ (regExLib.literalList "'_") @ specialChar @ [LITERAL #" ", LITERAL #"\t", LITERAL #"\n"]
  val alphabet_r = ALTERNATION (identifierList);
 
- val Any = STAR alphabet_r;
```

Comment:
Just a question. Why was this change necessary?

Reply:
There was some kind of clash with something else called "Any". Ugly, I know, please suggest a better fix if you have anything in mind.

Reply:
No, I think we should leave it at that. I was just curious how this can clash because I thought it would need to be a keyword rather than an identifier for this to happen.